### PR TITLE
VIDLA-4692

### PR DESCRIPTION
**Pull request by ynisman**
Fix for VIDLA-4692 (MAC Chrome - multi-player page (non-embedded) - several VIDEO JS type errors are thrown). Fix done on test page.

### CODE REVIEW

File: tests/e2e/testPages/prebid-body-multi-players.html [hunk 1]
```diff
@@ -101,7 +101,7 @@
 
 	    function pageNotification(type, data) {
 			if (type === 'event') {
-	    		appendStatistics('Event: ' + data);
+	    		appendStatistics('Event: ' + data, 1);
```
Comment (vpitts) [inline tests/e2e/testPages/prebid-body-multi-players.html]: can we add a comment stating the meaning of the "1" argument

File: tests/e2e/testPages/prebid-body-multi-players.html [hunk 2]
```diff
@@ -110,7 +110,7 @@
 	    
 	    function pageNotification2(type, data) {
 			if (type === 'event') {
-	    		appendStatistics('Event: ' + data);
+	    		appendStatistics('Event: ' + data, 2);
```
Comment (vpitts) [inline tests/e2e/testPages/prebid-body-multi-players.html]: likewise - we need a comment stating the purpose of the numerical digit

File: tests/e2e/testPages/prebid-body-multi-players.html (hunk 1 above)
Comment (ynisman) [reply tests/e2e/testPages/prebid-body-multi-players.html]: added comment

File: tests/e2e/testPages/prebid-body-multi-players.html (hunk 2 above)
Comment (ynisman) [reply tests/e2e/testPages/prebid-body-multi-players.html]: added comment

**Pull request merged**